db/sql/0001.sql

```diff
@@ -8,12 +8,29 @@ BEGIN
 END;
 $$ language 'plpgsql';
 
+CREATE TABLE IF NOT EXISTS ref_cluster (
```

Comment:
just for my knowledge is this a form of a migration?

I honestly like this more than using something like knex.js or sequelize

Reply:
It can read that way. Since I don't expect the schema to exists `CREATE TABLE` would be fine. Its just a habit in case its run multiple times (it won't fail) but it also won't make any changes. Migrations would generally have a `migrations` table in the DB and as the last line in the SQL insert the name of the script into the table to note it was already run.